modules/nf-core/cdhit/cdhit/main.nf

```diff
@@ -0,0 +1,40 @@
+process CDHIT {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::cd-hit=4.8.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/cd-hit%3A4.8.1--h5b5514e_7':
+        'quay.io/biocontainers/cd-hit:4.8.1--h5b5514e_7' }"
+
+    input:
+    tuple val(meta), path(sequences)
+
+    output:
+    tuple val(meta), path("*.fasta")    ,emit: fasta
+    tuple val(meta), path("*.clstr")    ,emit: clusters
+    path "versions.yml"                 ,emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def threshold = params.cdhit_threshold ?: 0.9
+    def wordsize = params.cdhit_wordsize ?: 5
```

Comment:
These two parameters (and other non-file ones) are to be passed via `task.ext.args`. Number 1 at https://nf-co.re/docs/contributing/modules#general